modules/imaging_browser/php/NDB_Form_imaging_browser.class.inc

```diff
@@ -51,7 +51,7 @@ class NDB_Form_Imaging_Browser extends NDB_Form
         // allow only to view own site data
         $timePoint =& TimePoint::singleton($_REQUEST['sessionID']);
 
-        return ($user->hasPermission('imaging_browser_view_allsites') || ((in_array($timePoint->getData('CenterID'), $user->getData('CenterIDs'))) && $user->hasPermission('imaging_browser_view_site')));
+        return ($user->hasPermission('imaging_browser_view_allsites') || ((in_array($timePoint->getData('CenterID'), $user->getData('CenterIDs'))) && $user->hasPermission('imaging_browser_view_site')) || $user->hasPermission('imaging_browser_phantom_data'));
```

Comment:
I'm not sure how this line is passing PHPCS, but can you format this into multiple lines so that the logical groupings are easier to follow?